services/openshift/scripts/openshift

```diff
@@ -136,3 +128,11 @@ fi
 # Now we start the server pointing to the prepared config files
 echo "[INFO] Starting OpenShift server"
 start_openshift '' --master-config="${master_config}" --node-config="${node_config}"
+wait_for_openshift_api
```

Comment:
This will not work because after L-130 nothing is going to run in openshift service because openshift container is not going to start in daemon mode. I suggest to put L133-138 before L-130 to make sure this execute.